### PR TITLE
Some comments and tests for conv and decimal packages

### PR DESCRIPTION
**Pull request by pcherednichenko**

### CODE REVIEW

File: conv/main.go [hunk 1]
```diff
@@ -6,10 +6,12 @@ const (
 	places int32 = 6
 )
 
+// FloatToDecimal convert float to decimal
 func FloatToDecimal(input float64) decimal.Decimal {
 	return decimal.NewFromFloat(input).Truncate(places)
 }
 
+// StringToDecimal convert number in string to decimal
```
Comment (im-kulikov) [inline conv/main.go:14]: mb `string number`? what you think?

File: conv/main.go [hunk 2]
```diff
@@ -6,10 +6,12 @@ const (
 	places int32 = 6
 )
 
+// FloatToDecimal convert float to decimal
```
Comment (im-kulikov) [inline conv/main.go]: ...with default precision

File: conv/main.go (hunk 1 above)
Comment (pcherednichenko) [reply conv/main.go:14]: nope

Comment (im-kulikov) [reply conv/main.go:14]: hm.. ok

**Review by kpeo — APPROVED** [review]

**Review by jenchik — APPROVED** [review]

**Pull request merged**